### PR TITLE
feat: add Sejal-16

### PR DESCRIPTION
**Pull request by Sejal-16**
Add  me 😀

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/301"></a>

### CODE REVIEW

File: public/data/Sejal-16.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Sejal Zambare",
+    "bio": " Hi there 😃,I am in my pre-final year of computer science engineering. I am passionate about web development and competitive coding. I am now looking for open source opportunities.Feel free to connect with me.",
```
Comment (schmelto) [inline public/data/Sejal-16.json:3]:
```suggestion
    "bio": " Hi there 😃,I am in my pre-final year of computer science engineering. I am passionate about web development and competitive coding. I am now looking for open source opportunities. Feel free to connect with me.",
```

Comment (Sejal-16) [reply public/data/Sejal-16.json]: Thanks for the suggestion :) Appreciate for reviewing.

**Review by schmelto — APPROVED** [review]
lgtm 
image

File: public/data/Sejal-16.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Sejal Zambare",
+    "bio": " Hi there 😃,I am in my pre-final year of computer science engineering. I am passionate about web development and competitive coding. I am now looking for open source opportunities. Feel free to connect with me.",
```
Comment (schmelto) [inline public/data/Sejal-16.json:3]:
```suggestion
    "bio": " Hi there 😃, I am in my pre-final year of computer science engineering. I am passionate about web development and competitive coding. I am now looking for open source opportunities. Feel free to connect with me.",
```

**Pull request merged**